torch/export/_trace.py

```diff
@@ -1684,17 +1735,6 @@ def _export_for_training(
     fake_mode = export_artifact.fake_mode
     module_call_specs = export_artifact.module_call_specs
 
-    # TODO(tmanlaibaatar) Not sure why i need this, but need to re-normalize it.
```

Comment:
What happened here?

Reply:
Ah, calling make_fx(record_module_stack=True) goes through ModuleStackTracer, which will do this when the node is created